src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/settings/SettingsPluginConfigurable.kt

```diff
@@ -68,9 +68,23 @@ class SettingsPluginConfigurable : Configurable {
         settingsState.buildPath = settingsComponent!!.buildPath
         settingsState.buildCommand = settingsComponent!!.buildCommand
         settingsState.telemetryEnabled = settingsComponent!!.telemetryEnabled
+        if (checkEmptyTelemetryPath(settingsState)) return
         settingsState.telemetryPath = settingsComponent!!.telemetryPath
     }
 
+    /**
+     * Check if the telemetry path is empty when telemetry is enabled.
+     * If empty, then sets to previous state. Else, keep the new one.
+     */
+    private fun checkEmptyTelemetryPath(settingsState: TestGenieSettingsState): Boolean {
+        if (settingsComponent!!.telemetryEnabled && settingsComponent!!.telemetryPath.isEmpty()) {
```

Comment:
Can you maybe change `isEmpty` to `isBlank`? Because blank strings are allowed right now

Reply:
ok